frontend/src/employee-mobile-frontend/child-attendance/actions/MarkPresent.tsx

```diff
@@ -68,7 +68,8 @@ export default React.memo(function MarkPresent() {
     return attendances.length > 0 ? attendances[0].departed : null
   }, [attendanceStatuses, childId])
 
-  const isValidTime = useCallback(() => {
+  const isValidTimeCheck = useCallback(() => {
+    if (!isValidTime(time)) return false
```

Comment:
`isValidTime()` should be moved to `LocalTime.tryParse`, otherwise this same bug may appear somewhere else, too.